tests/app/main/views/service_settings/test_service_settings.py

```diff
@@ -3890,12 +3890,17 @@ def test_GET_email_branding_upload_logo(client_request, service_one, query_param
     form = page.select_one("form")
     submit_button = form.select_one("button")
     file_input = form.select_one("input")
+    page_links = page.select("a")
+    skip_link = next(filter(lambda link: link.text == "I do not have a file to upload", page_links), None)
```

Comment:
Something like ‘select the last link in the `<main>` element’ feels more readable and will give more useful exceptions if something changes
```suggestion
    skip_link = page.select('main a')[-1]
    assert skip_link.text == "I do not have a file to upload"
```